src/lib/SearchMap.js

```diff
@@ -259,7 +272,7 @@ export default class SearchMap {
     polarCoordinatesFromExtent(projection, west, south, east, north) {
         let box = multisegmentBoxFromExtent(west, south, east, north);
         let globalToPolar = getTransform('EPSG:4326', projection);
-        return _.chunk(globalToPolar(box), 2);
+        return [_.chunk(globalToPolar(box), 2)];
```

Comment:
Is the array response from polarGeometryFromGlobalExtent still necessary, given that an array is now returned here? (See line 298.)

Reply:
Let me take a look, I may be getting lost in all the nested array stuff...